torch/csrc/autograd/python_variable_indexing.cpp

```diff
@@ -75,6 +75,10 @@ static Variable applySlice(const Variable& self, int64_t dim, PyObject* slice, b
     // TODO: implement negative step
     throw ValueError("negative step not yet supported");
   }
+  if (start >= stop) {
+    // TODO: implement 0-length slicing
```

Comment:
can you be more elaborate here, i.e., mentioning how we don't have proper 0-sized dim support?